bliss/encoder/image_normalizer.py

```diff
@@ -106,17 +130,16 @@ def get_input_tensor(self, batch):
         if self.use_clahe:
             renormalized_img = self.clahe(raw_images, self.clahe_min_stdev)
             inputs.append(renormalized_img)
-            inputs[0] = self.clahe(backgrounds, self.clahe_min_stdev)
+            if not self.asinh_params:
```

Comment:
This is also surprising behavior (from the perspective of user). If the user sets `use_clahe` to true, they would expect bliss to indeed use clahe normalization.

Reply:
have fixed it